reporter/src/main/kotlin/reporters/StaticHtmlReporter.kt

```diff
@@ -652,6 +653,26 @@ private fun DIV.licenseLink(license: String) {
     }
 }
 
+private fun DD.licensesLink(expression: SpdxExpression) {
+    val licenses = expression.decompose().map { it.toString() }
+
+    val licenseComponents = "$expression".split(" ")
```

Comment:
But we will never be able to find the text for exceptions, as `getLicensePath()` does not also look in `/resources/exceptions`.

Reply:
Good point about the `exceptions` I wasn't aware of it. 
I don't think you got it quite right. I am splitting `expression` which as the complete SpdxExpression. This can, e.g., be an `SpdxCompoundExpression`. I am not splitting the individual licenses. This is to split the licenses from the operators, creating a list of licenses and operators.
I see that with the exceptions this might not be the best approach.